storybook/main.js

```diff
@@ -8,6 +8,7 @@ const stories = [
 	'../packages/block-editor/src/**/stories/*.js',
 	'../packages/components/src/**/stories/*.js',
 	'../packages/icons/src/**/stories/*.js',
+	'../packages/edit-site/src/**/stories/*.js',
```

Comment:
Maybe I'm missing something, but why is this change needed in this PR?

Reply:
In order for the global styles story to be included, otherwise it's not.